### PR TITLE
Migrate icons to material design icons

### PR DESCRIPTION
**Pull request by GretaD**
ref https://github.com/nextcloud/groupware/issues/38
Please review only the icons on the third column, just the form. No action menus or styling. 
I fixed a bit the styling to make it better to review the icons, otherwise you cannot tell which label belongs to which icon.
This can be merged with the styling because it is valid, but needs more polishing that i will do in a separate pr

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Nice solution!

File: src/components/Properties/PropertyTitleIcon.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!--
+  - @copyright Copyright (c) 2022 Greta Doci <gretadoci@gmail.com>
+  -
+  - @author JGreta Doci
```
Comment (st3iny) [inline src/components/Properties/PropertyTitleIcon.vue:4]:
```suggestion
  - @author Greta Doci
```

**Review by st3iny — APPROVED** [review]
Tested and works. Code looks good too.

I like your approach of a combined icon component.

File: src/components/Properties/PropertyTitleIcon.vue (hunk 1 above)
Comment (GretaD) [reply src/components/Properties/PropertyTitleIcon.vue:4]: thats my new user name

**Pull request merged**